src/qt/addresstablemodel.cpp

```diff
@@ -288,7 +288,7 @@ QVariant AddressTableModel::headerData(int section, Qt::Orientation orientation,
 Qt::ItemFlags AddressTableModel::flags(const QModelIndex &index) const
 {
     if(!index.isValid())
-        return nullptr;
+        Qt::NoItemFlags;
```

Comment:
```suggestion
        return Qt::NoItemFlags;
```

Reply:
Not sure how I forgot the return statement.  Nice catch!